src/viam/api/CMakeLists.txt

```diff
@@ -113,6 +113,20 @@ if (VIAMCPPSDK_USE_DYNAMIC_PROTOS)
     )
   endif()
 
+  # buf generate insists that protoc is available in PATH, so we handle the situation
```

Comment:
Is this true only if we are using OFFLINE generation?

Reply:
Good question, will experiment